client/api/omni/specs/omni.proto

```diff
@@ -158,6 +158,11 @@ message MachineStatusSpec {
     string config = 1;
   }
 
+  message TalosMachineStatus {
+    machine.MachineStatusEvent status = 1;
```

Comment:
This type actually differs from the Talos's `MachineStatus` resource, despite being functionally the same, but I chose to reuse this type for simplicity, as it is already imported in this proto file.